### PR TITLE
Lots of fixes

### PR DESCRIPTION
**Pull request by andrewmp1**
- Fixed picking up user sessions when refreshing page
- Fixed organizations list and create
- Removed cruft from existing rails controllers
- Added a catch all route
- Fixed link to github repo.

### CODE REVIEW

File: app/assets/javascripts/controllers/edit_term_controller.js.coffee [hunk 1]
```diff
@@ -1,4 +1,4 @@
-YJ.EditTermController = Ember.ObjectController.extend(
+YJ.TermEditController = Ember.ObjectController.extend(
```
Comment (oldfartdeveloper) [inline app/assets/javascripts/controllers/edit_term_controller.js.coffee:1]: Thanks, agree with the naming change.

File: app/assets/javascripts/controllers/organizations_new_controller.js.coffee [hunk 2]
```diff
@@ -1,12 +1,9 @@
 YJ.OrganizationsNewController = Ember.ObjectController.extend(
   content: null
```
Comment (oldfartdeveloper) [inline app/assets/javascripts/controllers/organizations_new_controller.js.coffee:2]: Should this attribute 'content' remain?  Or should we delete it?  I'm hearing mixed signals from the core team.

File: app/assets/javascripts/controllers/terms_new_controller.js.coffee [hunk 3]
```diff
@@ -1,5 +1,6 @@
-YJ.NewTermController = Ember.Controller.extend(
-  organizationBinding: 'YJ.router.organizationController.content'
+YJ.TermsNewController = Ember.Controller.extend(
+  needs: ['organization']
```
Comment (oldfartdeveloper) [inline app/assets/javascripts/controllers/terms_new_controller.js.coffee:2]: Urgh.  Wouldn't this be better assigned in the route?  Yehuda and I agree on this one (or at least we did at RailsConf :confused: 

I mean this as a discussion item.

**Comment by oldfartdeveloper** [comment]
Nice job!  Thanks for all your work.  :thumbsup:

File: app/assets/javascripts/controllers/organizations_new_controller.js.coffee (hunk 2 above)
Comment (andrewmp1) [inline app/assets/javascripts/controllers/organizations_new_controller.js.coffee:2]: I've heard they are deprecating content and want to use model.  I've found situations where I used content and the magic worked funny. 

Respectfully,

Andrew Purdy

**Pull request merged**